libs/prov/prov_k8s_cortx_deploy.py

```diff
@@ -1278,29 +1255,35 @@ def test_deployment(self, sns_data, sns_parity,
             sol_file_path = resp[1]
             system_disk_dict = resp[2]
             LOGGER.info("Step to Perform Cortx Cluster Deployment")
-            resp = self.deploy_cortx_cluster(sol_file_path, master_node_list,
-                                             worker_node_list, system_disk_dict,
-                                             self.git_script_tag, namespace)
+            deploy_resp = self.deploy_cortx_cluster(sol_file_path, master_node_list,
+                                                    worker_node_list, system_disk_dict,
+                                                    self.git_script_tag, namespace)
+            LOGGER.debug("DEPLOY resp %s", deploy_resp)
+
             if len(namespace) >= self.deploy_cfg["max_size_namespace"] or \
                     bool(re.findall(r'\w*[A-Z]\w*', namespace)):
                 LOGGER.debug("Negative Test Scenario")
-                assert_utils.assert_false(resp[0], resp[1])
-            else:
+                assert_utils.assert_false(deploy_resp[0], deploy_resp[1])
+
+            # Run status-cortx-cloud.sh script to fetch the status of all resources.
+            if deploy_resp[0]:
+                LOGGER.info("Validate cluster status using status-cortx-cloud.sh")
+                resp = self.validate_cluster_status(master_node_list[0],
+                                                    self.deploy_cfg["k8s_dir"])
                 assert_utils.assert_true(resp[0], resp[1])
-            matches = [re.compile(pat) for pat in self.patterns]
-            if not (m.match(resp[1]) for m in matches):
-                LOGGER.info("Step to Check  ALL service status")
-                time.sleep(self.deploy_cfg["sleep_time"])
-                service_status = self.check_service_status(master_node_list[0])
-                LOGGER.info("service resp is %s", service_status)
-                assert_utils.assert_true(service_status[0], service_status[1])
-                row.append(service_status[-1])
-                if self.deployment_type != self.deploy_cfg["deployment_type_data"]:
-                    if self.cortx_server_image:
-                        resp = self.verfiy_installed_rpms(master_node_list,
-                                                          common_const.RGW_CONTAINER_NAME,
-                                                          self.deploy_cfg["rgw_rpm"])
-                        assert_utils.assert_true(resp[0], resp[1])
+                if not deploy_resp[1]:
+                    LOGGER.info("Step to Check  ALL service status")
+                    time.sleep(self.deploy_cfg["sleep_time"])
+                    service_status = self.check_service_status(master_node_list[0])
+                    LOGGER.info("service resp is %s", service_status)
```

Comment:
@pragamj As discussed, please log the hctl status --json output post deployment.

Reply:
please check the logs its being logged post deployment from all the data ,server pods.